bamboo/bootstrap-tf-deployment.sh

```diff
@@ -61,7 +61,7 @@ if [[ $NGAP_ENV = "SIT" ]]; then
   CMA_LAYER_VERSION=11
```

Comment:
Todo:  This was removed from SIT yesterday, I need to re-add and update it in SIT.

Reply:
4249b8357